sphinx/util/docutils.py

```diff
@@ -182,10 +183,41 @@ def using_user_docutils_conf(confdir: Optional[str]) -> Generator[None, None, No
             os.environ['DOCUTILSCONFIG'] = docutilsconfig
 
 
+@contextmanager
+def du19_footnotes() -> Generator[None, None, None]:
+    def visit_footnote(self, node):
+        label_style = self.settings.footnote_references
+        if not isinstance(node.previous_sibling(), type(node)):
+            self.body.append(f'<aside class="footnote-list {label_style}">\n')
+        self.body.append(self.starttag(node, 'aside',
```

Comment:
Is it okay to output `<aside>` tag forcedly? For example, sphinx_rtd_theme does not support docutils-0.18. And it expects not to use `<aside>` for footnotes. So this will break renderings of the theme.
https://github.com/readthedocs/sphinx_rtd_theme/blob/master/setup.py#L122

Reply:
Ahh, yes this should be also gated on DU 18 or above.

A